std/algorithm/iteration.d

```diff
@@ -860,6 +860,7 @@ template each(alias pred = "a")
     import std.meta : AliasSeq;
     import std.traits : Parameters;
 
+private:
```

Comment:
Technically this is a "breaking" change, but IIRC non-documented members can be set to `private` without deprecation path is the user can only know about them by reading the code on his own and no guarantees were given.